packages/paste-core/components/textarea/src/TextArea.tsx

```diff
@@ -60,6 +60,9 @@ const TextAreaElement = styled(TextareaAutosize)<TextAreaProps>((props) =>
     '&:disabled': {
       color: props.variant === 'inverse' ? 'colorTextInverseWeaker' : 'colorTextWeaker',
       cursor: 'not-allowed',
+      // Fixes value color in Safari
+      '-webkit-text-fill-color': props.variant === 'inverse' ? 'colorTextInverseWeaker' : 'colorTextWeaker',
```

Comment:
Just curious, why does this fix the issue? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-fill-color), if this isn't provided the `color` will be picked up anyways: 
> If this property is not set, the value of the color property is used.

Reply:
@TheSisb unsure as well. I read that in the MDN page also, so using `color` should just work. I'm wondering if it's a bug in Safari, or they just decided to ignore the spec.